ra/ra.go

```diff
@@ -1376,24 +1376,36 @@ func (ra *RegistrationAuthorityImpl) checkCertificatesPerFQDNSetLimit(ctx contex
 		return nil
 	}
 
-	issuanceTimestampsInWindow, err := ra.SA.FQDNSetTimestampsForWindow(ctx, &sapb.CountFQDNSetsRequest{
+	issuance, err := ra.SA.FQDNSetTimestampsForWindow(ctx, &sapb.CountFQDNSetsRequest{
```

Comment:
ra.go imports the issuance package, so this variable name shadows that. Maybe `prevIssuances` or something like that?